servicenow.config.js

```diff
@@ -10,18 +10,18 @@ const servicenowConfig = {
    * Current configuration does not produce CSS files
    * CSS code will be embedded into javascript files
    */
-  JS_API_PATH: 'api/x_744337_vue_app/container/js/',
+  JS_API_PATH: 'api/x_g_dis_atat/vue_app_container/js/',
   /**
    * ServiceNow path to GET resource which serves
    * Image files (png, jpg, gif)
    * SVG files will be embedded into javascript files
    */
-  IMG_API_PATH: 'api/x_744337_vue_app/container/img/',
+  IMG_API_PATH: 'api/x_g_dis_atat/vue_app_container/img/',
   /**
    * ServiceNow path to GET resource which serves
    * other files, like fonts etc.
    */
-  ASSETS_API_PATH: 'api/x_744337_vue_app/container/other_assets/',
+  ASSETS_API_PATH: 'api/x_g_dis_atat/vue_app_container/other_assets/',
```

Comment:
I wonder if `REST_API_PATH` could contain `api/x_g_dis_atat/vue_app_container`
and then prepend that on each of `JS_API_PATH`, `IMG_API_PATH`, and `ASSETS_API_PATH`.

I don't see that `REST_API_PATH` is referenced elsewhere, and this seems like its intended use.

Reply:
I've done the above in #967, however I had to pull `REST_API_PATH` out of the enclosing object to be able to reference it.